Documentation for Discussion migration to 6.1.

mauritsvanrees opened the pull request:
This is for the PLIP that I have just merged:
https://github.com/plone/Products.CMFPlone/issues/3958

I changed the existing section title from "Discussion is disabled by default" to "Discussion is a core add-on". Discussion was always disabled by default, so this would not be a change.

Note that the Discussion control panel is still missing from Volto: https://github.com/plone/volto/issues/29 But the upgrade to 6.1 does not change anything to this.


<!-- readthedocs-preview plone6 start -->
----
📚 Documentation preview 📚: https://plone6--1696.org.readthedocs.build/

<!-- readthedocs-preview plone6 end -->

mauritsvanrees committed:
Documentation for Discussion migration to 6.1.

This is for the PLIP that I have just merged:
https://github.com/plone/Products.CMFPlone/issues/3958

I changed the existing section title from "Discussion is disabled by default" to "Discussion is a core add-on".
Discussion was always disabled by default, so this would not be a change.

Note that the Discussion control panel is still missing from Volto: https://github.com/plone/volto/issues/29
But the upgrade to 6.1 does not change anything to this.

stevepiercy commented on docs/backend/upgrading/version-specific-migration/upgrade-to-61.md:
```diff
@@ -90,17 +90,29 @@ The goal of turning more of the current core packages into core add-ons is to ma
 1.  Restart the Plone backend to load `plone.app.discussion`.
-1.  Enable the {guilabel}`Discussion` add-on in the Plone control panel under {menuselection}`Site Setup --> General`.
+1.  Enable the {guilabel}`Discussion Support` add-on in the {guilabel}`Add-ons` control panel under {menuselection}`Site Setup --> General`.
+1.  If you use Plone Classic UI, you can then use the {guilabel}`Discussion` control panel to further configure this feature, for example for enabling comment moderation.
```
```suggestion
1.  If you use Plone Classic UI, you can then use the {guilabel}`Discussion` control panel to further configure this feature, for example, to enable comment moderation.
```

stevepiercy committed:
Review from @stevepiercy

Mostly style guide stuff, but I added a clause to the final bullet point that needs review from @mauritsvanrees:

```
Add the `plone.app.discussion` package to your dependencies, and run the migration again.
```

stevepiercy reviewed (approved):
Mostly style guide stuff, but I added a clause to the final bullet point that needs review from @mauritsvanrees:

```
Add the `plone.app.discussion` package to your dependencies, and run the migration again.
```

stevepiercy commented:
The failing linkcheck can be ignored, as it was a temporary outage.

mauritsvanrees commented:
> Mostly style guide stuff, but I added a clause to the final bullet point that needs review from @mauritsvanrees:
> 
> ```
> Add the `plone.app.discussion` package to your dependencies, and run the migration again.
> ```

Thanks for polishing it up.  The new wording is fine.
I merge.
